test/functional/dataElements/xdmObject.spec.js

```diff
@@ -54,14 +54,7 @@ const expectSettingsToContainData = async data => {
   // new tests), the schema version in Platform changes, which would make our
   // tests fail if the version we were asserting were hard-coded in the test.
   await t.expect(actualSettings.schema.version).match(/^\d+\.\d+$/);
-  await t
-    .expect(actualSettings.data)
-    .eql(
-      data,
-      `Expected data: ${JSON.stringify(data)} Actual data: ${JSON.stringify(
-        actualSettings.data
-      )}`
-    );
```

Comment:
The prior version of TestCafe we were using wasn't very good at showing the diff between two objects when we tried to compare them using `.eql`. It would look like this:

![image](https://user-images.githubusercontent.com/210820/97529761-3458c400-1976-11eb-9cef-717706827ed9.png)

which wasn't very useful because it didn't show us which property was different. This is why I had to output both serialized objects like this when the test failed so that the developer could at least compare the objects visually. The latest version added an enhancement that shows a better diff of the object:

![image](https://user-images.githubusercontent.com/210820/97529836-60744500-1976-11eb-8727-154ce32baa0a.png)

So now we no longer need to log the stringified objects. 🎉